deepchem/molnet/load_function/bbbc_datasets.py

```diff
@@ -29,8 +31,15 @@ def create_dataset(self) -> Dataset:
         if not os.path.exists(labels_file):
             dc.utils.data_utils.download_url(url=BBBC1_LABEL_URL,
                                              dest_dir=self.data_dir)
+        labels_table = pd.read_csv(labels_file, delimiter="\t")
```

Comment:
The documents for this say that the true labels are the averages of the cells counts done by the 2 professional human counters.